devices.js

```diff
@@ -11179,6 +11179,19 @@ const devices = [
         description: 'ZigBee 3.0 LED-controller, 4 channel 5A, RGBW LED',
         extend: preset.light_onoff_brightness_colortemp_colorxy,
     },
+    {
+        zigbeeModel: ['HK-ZD-RGBCCT-A', '511.000'],
+        model: '511.000',
+        vendor: 'Iluminize',
+        description: 'ZigBee 3.0 Universal LED-controller, 5 channel 4A, RGBCCT LED',
+        extend: preset.light_onoff_brightness_colortemp_colorxy,
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint, logger) => {
```

Comment:
Why was the configure added here?

Reply:
To be honest I didn't even realize of that. As described, I just copied configuration from 511.040 which shares same capabilities and Zigbee 3.0 compatibility and AFAIK only changes the number of channels and amperage. But supports same things.

Reply:
The `511.040` doesn't have this so I will remove it. Thanks!